client/src/components/Home.jsx

```diff
@@ -55,13 +60,13 @@ const Home = ({ setIsLoading, isMobile, isLoggedIn, isLoading, setIsLoggedIn, sh
   const getAllTodos = (user) => {
     axios.get('http://localhost:3000/todoList/info', { params: { email: userEmail } })
       .then((result) => {
+          console.log('result: ', result);
           setMyEvents(result.data.results[0].calendars[0].categories);
         })
         .catch(err => console.error(err));
   }
 
   // POST '/todoList/:userEmail' -> Adding or Upserting a "todoList item"
-  //modified to use actual user email
   const addTodo = (todo) => {
     axios.post('http://localhost:3000/todoList/item', { params: { userEmail: userEmail }, data: todo })
```

Comment:
doesn't this need to be `{ params: { email: userEmail }, data: todo })` ?

Reply:
NVM!

Reply:
Per our discussion, different line of code--though I think the email is extraneous in the code above.